src/services/edusharing/hooks/merlin.hooks.js

```diff
@@ -1,6 +1,14 @@
 const { authenticate } = require('@feathersjs/authentication');
 const { disallow } = require('feathers-hooks-common');
 const { NotFound } = require('@feathersjs/errors');
+const { Configuration } = require('@schul-cloud/commons');
+
+const checkMerlinConfiguration = (hook) => {
+	if (!Configuration.get('ES_MERLIN_ENABLED')) {
+		throw new Error('Merlin is not enabled');
```

Comment:
How is this error handled and where it its catch?

Reply:
It throws when it's not enabled, causing the url not to be created. If the merlinservice fails to generate the url, nuxt will provide with a placeholder instead.

Reply:
releay and no output at the logs for it?